src/staged_policy.jl

```diff
@@ -0,0 +1,44 @@
+
+"""
+StagedPolicy
+
+Policy for finite horizon POMDP solvers.
+Wraps multiple Policies, one for each stage.
+"""
+struct StagedPolicy{M<:FiniteHorizonPOMDPs.FHWrapper, P<:Policy}<:Policy
```

Comment:
```suggestion
struct StagedPolicy{M<:Union{POMDP,MDP}, P<:Policy}<:Policy
```
It seems like this should support any POMDP for which `HorizonLength` is finite.